tests/cli/dorq/ui/test_prompts.py

```diff
@@ -0,0 +1,84 @@
+################################################################################
+# © Copyright 2023 Zapata Computing Inc.
+################################################################################
+
+"""
+Tests for CLI prompters.
+"""
+
+from unittest.mock import Mock
+
+import inquirer  # type: ignore # noqa
+import pytest
+
+from orquestra.sdk._base.cli._dorq._ui._prompts import SINGLE_INPUT, Prompter
+from orquestra.sdk.exceptions import UserCancelledPrompt
+
+prompter = Prompter()
```

Comment:
Suggestion: Might be better to do it as class--scoped thing.

Reply:
Yes, it absolutely would. However, as the docstring of Prompter says:
```
Given it's our system's boundary it would be nice to write tests that perform real 
stdin/stdout IO, but simulating arrow key strokes is very tricky!
```
As such, I think just testing the new logic for this PR and leaving proper testing of the Prompter to https://zapatacomputing.atlassian.net/browse/ORQSDK-834 is a sensible compromise.